lighthouse-extension/gulpfile.js

```diff
@@ -135,10 +146,15 @@ gulp.task('browserify-lighthouse', () => {
       bundle.ignore(require.resolve('../lighthouse-core/gather/connections/cri.js'));
 
       // Prevent the DevTools background script from getting the stringified HTML.
-      if (/lighthouse-background/.test(file.path)) {
+      if (isDevtools(file.path)) {
         bundle.ignore(require.resolve('../lighthouse-core/report/html/html-report-assets.js'));
       }
 
+      if (isDevtools(file.path) || isExtension(file.path)) {
+        // eslint-disable-next-line
```

Comment:
can we specify which rule to disable?

Reply:
sorry I don't need it anymore it was for testing console.log